ArduCopter/system.cpp

```diff
@@ -285,6 +285,10 @@ bool Copter::ekf_has_relative_position() const
         enabled = true;
     }
 #endif
+    Vector3f airspeed_vec_bf;
+    if (AP::ahrs().airspeed_vector_true(airspeed_vec_bf) && !dead_reckoning.timeout) {
```

Comment:
Given the intent of this check is to determine if we are dead reckoning and timed out, then it would be simpler to use:

    if (dead_reckoning.active && !dead_reckoning.timeout) {

Reply:
Great, thanks for that.  fixed.